### PR TITLE
M3-107 Pagination Controls.

### PR DESCRIPTION
**Pull request by rjstires**

### CODE REVIEW

**Comment by rjstires** [comment]
@alioso These are significantly larger than those found in the UI key; 
image

Additionally the first/last buttons appear to be slightly larger, creating a lip at the bottom;
image

Once these are addressed we can get this merged unless @asauber has feedback.

**Comment by asauber** [comment]
I agree with @rjstires comments. These are larger than the UI comps, and the first and last page buttons extend downward.

File: src/components/PaginationControls/PaginationControls.test.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    );
+
+    const prevPage = getFirstPageButton(wrapper);
+    const nextPage = getLastPageButton(wrapper);
+
+    expect(prevPage.exists()).toBeTruthy();
+    expect(nextPage.exists()).toBeTruthy();
+  });
+  
+  it('should render a left arrow when not on first page', () => {
```
Comment (asauber) [inline src/components/PaginationControls/PaginationControls.test.tsx]: This should read "it should not render a left arrow when on first page"

File: src/components/PaginationControls/PaginationControls.test.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        pages={mockPages}
+        currentPage={1}
+      />,
+    );
+    const prevPage = getFirstPageButton(wrapper);
+
+    expect(prevPage.exists()).toBeFalsy();
+  });
+  
+  it('should render a right arrow when not on last page', () => {
```
Comment (asauber) [inline src/components/PaginationControls/PaginationControls.test.tsx]: This should read "it should not render a right arrow when on last page"

**Comment by alioso** [comment]
@rjstires @asauber fixed test code, icon size and icon display

**Comment by rjstires** [comment]
@asauber Outside of the center ellipses `1,2,3,...7,8,9` we don't have anything in the FO or general acceptance criteria to describe the functionality in detail. Did you have a resource or example to can direct me to?

**Comment by rjstires** [comment]
@asauber @alioso ready for review please!

**Pull request merged**